### ISSUE TITLE
Translatable "for"

### ISSUE THREAD

**Issue by Madis0** [Fabulously-Optimized/vanilla-installer-rust#63]
An user suggested that the "for" in the version name could be translated, aka technically something like "{foversion} for {minecraftversion}".

The idea is interesting but that'd mean I cannot describe anything in the names, have to think...

**Comment by vgskye** [comment]
yeah, I'm not sure I particularly like the idea of throwing out the modrinth version names and constructing our own
